backend/src/reservations/reservations.controller.ts

```diff
@@ -46,6 +50,15 @@ export class ReservationsController {
       const reservationDto = await this.reservationsService.createReservation(
         reservation,
       );
+      const event = await this.eventService.findOneById(reservation.event_id);
+      const reservationMessage = await this.reservationsService.buildReservationMessage(
```

Comment:
Not sure if the controller action is the best place for sending the sms..  could this logic be behind the createReservation method? That way it would be easier to see what happens when a reservation is created, as everything would in one place. Now part of the logic is here in the controller, which is not good IMO..

Reply:
yes I agree :)